config_samples/ethereum/hoodi/hoodi_vaults_testnet_config.json

```diff
@@ -0,0 +1,428 @@
+{
+  "contracts": {
```

Comment:
I would prefer to have a more straightforward ordering tbh:
- proxy1
- proxy1 impl
- proxy2
- proxy2 impl

etc.